cmd/templ/generatecmd/main.go

```diff
@@ -251,17 +262,17 @@ func openURL(url string) error {
 	return browser.OpenURL(url)
 }
 
-func processSingleFile(ctx context.Context, fileName string, generateSourceMapVisualisations bool) error {
+func processSingleFile(ctx context.Context, fileName string, generateSourceMapVisualisations bool, outputFileName string) error {
```

Comment:
I think that if we're going to call one file name `outputFileName`, then the `fileName` variable would be better named as `inputFileName`.

Also, I think the order of the parameters should be shifted.

```suggestion
func processSingleFile(ctx context.Context, inputFileName string, outputFileName string, generateSourceMapVisualisations bool) error {
```